app/services/gias/importer.rb

```diff
@@ -7,7 +7,10 @@ class Importer
     SCHOOL_LINKS_FILENAME = "links.csv"
 
     def fetch
-      import_only? ? fetch_and_import_only : fetch_and_update
+      Metadata::Manager.skip_metadata_updates do
+        import_only? ? fetch_and_import_only : fetch_and_update
+      end
+      Metadata::Handlers::School.refresh_all_metadata!(async: true, track_changes: true)
```

Comment:
I dont think we should track changes here, as we expect some changes.
We should track changes for the nightly runs so we receive sentry events for any changes